arm/ripper/identify.py

```diff
@@ -20,17 +19,32 @@
 from arm.ui import utils as ui_utils
 
 
+def check_if_mounted(mounted):
+    """
+    Function to check if mounting disc was success
+     anything but 0 means we failed to mount disc
+    """
+    success = False
+    if mounted == 0:
+        logging.info("Mounting disc was successful")
+        success = True
+    else:
+        logging.error("Mounting failed! Rip might have problems")
+    return success
+
+
 def identify(job):
     """Identify disc attributes"""
     logging.debug("Identify Entry point --- job ----")
     logging.info(f"Mounting disc to: {job.mountpoint}")
     if not os.path.exists(str(job.mountpoint)):
         os.makedirs(str(job.mountpoint))
-
-    os.system("mount " + job.devpath)
-
-    # Check with the job class to get the correct disc type
-    job.get_disc_type(utils.find_file("HVDVD_TS", job.mountpoint))
+    # Check and mount drive - log error if failed
+    mounted = check_if_mounted(os.system("mount " + job.devpath))
+    # get_disc_type() checks local files, no need to run unless we can mount
+    if mounted:
```

Comment:
Pretty sure this is a fatal error. You'll want the function to throw an error, and call it from a try/catch that'll kill the job if the error is caught

Reply:
Making this a fatal error would mean that running arm in container would never be possible without the `--priv` flag.

Reply:
The `--priv` that the user needs to get rips to mount the disks in the first place, yeah

Reply:
But its not needed.... The rip logs are attached. In the initial comment
Ripped and converted successfully without the `--privileged` flag.

Because ARM only mounts the disc to search for the disctype and to calculate the CRC64 which are only used by the dvd side of things.
The rest seem to rip without any issue, and the DVD only fails sometimes with the auto ident because its using the crc64. But if it can match omdb all is good.

Will need a lot more testing but removing the `--privileged` flag would be a big plus for the security side of things.

Reply:
Gotcha. I'm all for eliminating that risk if it's possible. Does `SYS_RAWIO` negate the performance loss too, or does it just fix the issue?

Reply:
I've only tested it with a USB drive, didn't have time for a full test. But it removed the performance issues.

It still ripped no problem without it, it just went from 13/15 mins fully ripped up to 23/25. Wouldn't make a difference using ARM normally (inserting disc and walking away) but for dev time it makes a huge diff in performance.

Reply:
Gotcha. I would say when you're ripping your entire library, 10 minutes per disk is game changing